fixtures/Alias.mad

```diff
@@ -0,0 +1,9 @@
+alias T = (a -> b)
+
+fn :: T -> a -> b
+fn = (cb, a) => (cb(a))
+
+export alias TT a = (a -> b)
+
+fn :: TT String -> a -> b
+fn = (cb, a) => (cb(a))
```

Comment:
Dope. I get it now.

Reply:
Another one, we can also name records now !

```
alias Player a = { position :: Position, object :: a, id :: Number }

findPlayer :: Number -> Player
// Instead of:
findPlayer :: Number -> { id :: Number }
```